rust_dev_preview/sdk-config/src/bin/set_retries.rs

```diff
@@ -76,7 +76,7 @@ async fn main() -> Result<(), Error> {
         config::Builder::from(&shared_config)
             // Set max attempts.
             // If tries is 1, there are no retries.
-            .retry_config(RetryConfig::standard().with_max_attempts(tries))
+            .retry_config(RetryConfig::default().with_max_attempts(tries))
```

Comment:
The latest released SDK has this as `standard()`: https://docs.rs/aws-smithy-types/0.49.0/aws_smithy_types/retry/struct.RetryConfig.html#method.standard

Odd that it's still `default()` on your system if the `Cargo.toml` is pointing at aws-sdk-rust's main branch. Something must be stale here.

Reply:
Looks like Clippy did fail this during our linting check.

Reply:
Weird, my local copy is mad at ::standard(). I'll cargo clean and try again.